ctx.go

```diff
@@ -1037,67 +1028,197 @@ func (c *DefaultCtx) Queries() map[string]string {
+	case int:
+		result, err := strconv.ParseInt(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int](int(result))
+	case int8:
+		result, err := strconv.ParseInt(q, 10, 8)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int8](int8(result))
+	case int16:
+		result, err := strconv.ParseInt(q, 10, 16)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int16](int16(result))
+	case int32:
+		result, err := strconv.ParseInt(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int32](int32(result))
+	case int64:
+		result, err := strconv.ParseInt(q, 10, 64)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int64](result)
+	case uint:
+		result, err := strconv.ParseUint(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint](uint(result))
+	case uint8:
+		result, err := strconv.ParseUint(q, 10, 8)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint8](uint8(result))
+	case uint16:
+		result, err := strconv.ParseUint(q, 10, 16)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint16](uint16(result))
+	case uint32:
+		result, err := strconv.ParseUint(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint32](uint32(result))
+	case uint64:
+		result, err := strconv.ParseUint(q, 10, 64)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint64](result)
+	case float32:
+		result, err := strconv.ParseFloat(q, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, float32](float32(result))
+	case float64:
+		result, err := strconv.ParseFloat(q, 64)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, float64](result)
+	case bool:
+		result, err := strconv.ParseBool(q)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, bool](result)
+	case string:
+		if q == "" && len(defaultValue) > 0 {
+			return defaultValue[0]
+		}
+		return assertValueType[V, string](q)
+	case []byte:
+		if q == "" && len(defaultValue) > 0 {
+			return defaultValue[0]
+		}
+		return assertValueType[V, []byte]([]byte(q))
```

Comment:
here's where the allocation in the benchmarks is coming from. you can avoid it using utils.UnsafeBytes
```suggestion
		return assertValueType[V, []byte](utils.UnsafeBytes(q))
```

Reply:
ok, i'll try it

Reply:
@nickajacks1 did I convert the whole to utils. Unsafe?

Reply:
just this one line

Reply:
only for []byte ?

Reply:
![image](https://github.com/gofiber/fiber/assets/17839029/56805c26-4c7c-4202-83a7-8a826c0e1079)
@nickajacks1 like this?

Reply:
@nickajacks1 i'm already push it